docker/docker-compose-server.yml

```
@@ -0,0 +1,30 @@
+version: '3.3'
+
+services:
+  postgres:
+    image: postgres:latest
+    volumes:
+    - /mnt/vol/postgresql:/var/lib/postgresql
+    environment:
+      POSTGRES_DB: sourcesdb
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    ports:
+    - 5432:5432
+  server:
+    container_name: codesearch_web_server
+    image: quay.io/aelve/codesearch-web-server:latest
+    volumes:
+    - /mnt/vol/data:/data
+    - /mnt/vol/index:/index
+    depends_on:
+    - postgres
+    command:
+      make serve
+    environment:
+      LOG_LEVEL: DEBUG
+      JAVA_OPTS: "-Dplay.filters.hosts.allowed.1=codesearch.aelve.com -Dplay.filters.hosts.allowed.2=cloud.digitalocean.com"
+      # In production you must override the APPLICATION_SECRET key
+      APPLICATION_SECRET: u?:IPZT:Cun;SP>edigHDs:cpUf2cY`<7D>b9lHg98^NsVGn]:eLCIjz1Tt_VyIv
```

Perhaps put it into quotes to avoid accidentally triggering weird YAML string rules.

```suggestion
      APPLICATION_SECRET: "u?:IPZT:Cun;SP>edigHDs:cpUf2cY`<7D>b9lHg98^NsVGn]:eLCIjz1Tt_VyIv"
```